glean-core/ios/Glean/Generated/GleanBaseline.swift

```diff
@@ -0,0 +1,29 @@
+// -*- mode: Swift -*-
+
+// AUTOGENERATED BY glean_parser.  DO NOT EDIT.
+// MANUALLY UPDATED. WILL BE AUTO-GENERATED IN THE FUTURE.
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+internal enum GleanBaseline {
```

Comment:
This should not be checked in, right?

Reply:
```
 // MANUALLY UPDATED. WILL BE AUTO-GENERATED IN THE FUTURE.
```

It needs to right now, because we're not auto-generating them yet.